scripts/simulate.py

```diff
@@ -0,0 +1,192 @@
+"""
+CLI for hera_sim
```

Comment:
Be a little bit more expansive in the description -- we might yet have other CLI scripts :-)

Reply:
Agreed, good catch.

Reply:
Updated the description. Lmk if you think I should make any changes.

Reply:
It's now `"""Re-imagining of the simulation module."""` right? I might go for something like "Simulate post-RIME effects via CLI"?

Reply:
Ah, no, that's currently the docstring for the `simulate` module where the `Simulator` is defined. I suppose it's bad practice to name a script the same thing as one of the modules. The new docstring for the CLI is much more descriptive than just a one-liner. (I'll definitely need to update the `simulate` module's docstring at some point, though.)